app/flatpak-cli-transaction.c

```diff
@@ -1082,6 +1084,46 @@ append_bus (GPtrArray  *talk,
     }
 }
 
+static void
+append_usb (GPtrArray *usb_array,
+            GKeyFile  *metadata,
+            GKeyFile  *old_metadata)
+{
+  gsize i, size = 0;
+  g_auto(GStrv) hidden_devs = NULL;
+  g_auto(GStrv) old_hidden_devs = NULL;
```

Comment:
Maybe call them `hidden_devices` and `old_hidden_devices` - my brain keeps imagining Flatpak developers playing hide and seek :ninja:

Reply:
make sense.